resolvers/hospital.js

```diff
@@ -68,19 +68,27 @@ const hospitalResolvers = {
                 args.id
             );
           }
-        });
-      await db.eventHospitals.destroy({
-        where: {
-          hospitalId: args.id,
-        },
-      });
-
-      return db.hospital.destroy({
-        where: {
-          id: args.id,
-        },
-      });
-    },
+        })
+        .then(() =>
+          db.eventHospitals.destroy({
+            where: {
+              hospitalId: args.id,
+            },
+          })
+        )
```

Comment:
destroy returns a Promise that always gets resolved with 1 on success or 0 otherwise (i.e. the promise is always “fulfilled” and won’t trigger a catch statement event when the deleting failed).

Note this means that there's no "error handling" for `db.eventHospitals.destroy` here.